### PR TITLE
Updated gnome cheatsheet. Added more shortcuts.

### PR DESCRIPTION
**Pull request by rahul-raturi**
I've included some more keyboard shortcuts in this cheat sheet. Also, I've updated the sourceURL as the previous sourceURL (https://help.gnome.org/users/gnome-help/stable/shell-keyboard-shortcuts.html.en) does not contains all of the shortcuts.

**Screenshots**
***Previously***
previous

***Updated***
present
-----
IA Page: https://duck.co/ia/view/gnome_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@rahul-raturi Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/json/gnome.json [hunk 1]
```diff
@@ -4,76 +4,180 @@
...
+            }
+        ],
+        "System": [
+            {
+                "val": "Show the run command prompt",
+                "key": "[Alt] [F2]"
+            },
+            {
+                "val": "Show the activities overview",
+                "key": "[Alt] [F1] / [Super]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gnome.json]: Please wrap the sub-alternatives in `()` to make it clear what the `/` separates: `[Alt] ( [F1] / [Super] )`

**Comment by moollaza** [comment]
@rahul-raturi Thanks a lot for contributing, we really appreciate it! I've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

File: share/goodie/cheat_sheets/json/gnome.json (hunk 1 above)
Comment (rahul-raturi) [inline share/goodie/cheat_sheets/json/gnome.json]: It is not like that. One is a key combination ([Alt] [F1]) and the other is a single key ([Super]). I guess it is looking correct.

**Comment by rahul-raturi** [comment]
**@moollaza**, it is not like that. One is a key combination ([Alt] [F1]) and the other is a single key ([Super]). I think it looks alright.

**Comment by moollaza** [comment]
> it is not like that. One is a key combination ([Alt] [F1]) and the other is a single key ([Super]). I think it looks alright.

In that case, they should be separated into their own KV pairs, and the description for one should indicate it's an alternative. With the current format its impossible to know if the `/` means "Alt F1" OR "Super" vs "Alt (F1 OR Super)".

**Comment by rahul-raturi** [comment]
@moollaza I guess the alternative isn't necessary for cheat sheet purpose. Moreover "super" is more frequently used, so I've removed the other one.

**Comment by moollaza** [comment]
@rahul-raturi apologies for the delay! I hadn't seen this. I'll deploy it to Beta right now and take another look..

**Comment by moollaza** [comment]
@rahul-raturi apologies for the delay!

this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Pull request merged**